mod/wikirate/spec/set/right/wikirate_article_spec.rb

```diff
@@ -25,14 +25,14 @@
         overview_name = "#{@company.name}+#{@topic.name}+#{Card[:wikirate_article].name}"
         article = Card.create :name=>overview_name, :type=>"basic",:content=>"hello world"
         html = article.format.render_content
-        expect(html).to have_tag("div",:with=>{:class=>"claim-tip"}) do
-          with_tag "textarea",:with=>{:id=>"citable_claim"},:text=>/#{@citation}/
+        expect(html).to have_tag("div",:with=>{:class=>"note-tip"}) do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.